### DISCUSSION TITLE
Granting a user write access to a particular package within organization

### DISCUSSION THREAD

**Discussion by mat-hek** [hexpm/hexpm#1266] [Q&A] [answered]
Hi there, we'd like to allow some 3rd party contributors to publish packages they maintain within the organization that we own. We need them not to have any permissions to other packages in the organization. All the packages are public. Does Hex support this?

**Comment by ericmj** [comment, marked as answer]
Yes. Since all packages are public you can add them to the organization with read permissions only and then add them as package owners to the individual packages.

**Reply by mat-hek** [reply]
Thanks! For the record: https://hexdocs.pm/hex/Mix.Tasks.Hex.Owner.html
